tcms/bugs/tests/test_models.py

```diff
@@ -37,3 +37,33 @@ def test_no_notification_if_assignee_not_set(self, send_mail):
         BugFactory(assignee=None)
 
         self.assertFalse(send_mail.called)
+
+
+class TestBugStatus(TestCase):
+    """Test the possible bug status changes.
+    When:
+    * Closing an openned bug.
+    * Reopenning a closed bug.
+    * Try to reopen an already openned bug.
+    * Try to close an already closed bug.
+    """
+
+    def test_close_an_openned_bug(self):
+        bug = BugFactory(status=True)
+        bug.status = False
+        self.assertFalse(bug.status)
```

Comment:
here you are only setting the value of a variable and asserting that. this will always pass and do not test the new functionality you have introduced. what we need is a test that will:
- create a new bug via `BugFactory` (this is OK)
- make a request to `/bugs/comment` with `action=reopen`/`action=close`
- fetch the Bug from the database
- assert that the bug's status has indeed been changed

Reply:
Hi @asankov, 

Thanks for the review. Actually that was just a placeholder, i will write the tests as per your suggestion. ;)

Reply:
Please check #1514